Library/Contracts.ts

```diff
@@ -294,45 +257,28 @@ export module Contracts {
             this.ver = 2;
             this.properties = {};
             this.measurements = {};
-
-            super();
         }
     }
 
-    export class RemoteDependencyTypes {
-        public static ApplicationInsights = "Application Insights";
-        public static Http = "Http";
-        public static Sql = "SQL";
-    }
-
     export class RemoteDependencyData extends Contracts.Domain {
         public ver:number;
         public name:string;
-        public kind:Contracts.DataPointType;
-        public value:number;
-        public count:number;
-        public min:number;
-        public max:number;
-        public stdDev:number;
-        public dependencyKind:Contracts.DependencyKind;
+        public id:string;
```

Comment:
Just curious, since the *.bond files contain description (and other) annotations why don't the generated .ts class/module files contain comments with this very useful data?